src/replicache.ts

```diff
@@ -279,6 +279,9 @@ export class Replicache<MD extends MutatorDefs = {}> {
   private _persistIsScheduled = false;
   private _recoveringMutations = false;
 
+  private readonly _disableLicensing: boolean = false;
```

Comment:
I don't know how your brains work but for mine "enableFoo" is so much easier to keep in mind than "disableFoo" which if on/true has the effect of turning something off.&lt;/shed&gt;

Reply:
I don't feel strongly at all about this.  

My choice of disable vs enable is based on the default.  If the default behavior is enabled, then the option is to disable it. 

For me
```
new Replicache({
  disableLicensing: true
});
```

scans faster than

```
new Replicache({
  enableLicensing: false 
});
```

Reply:
I don't care if it's internal API.

Reply:
I'm also in favor of enable over disable in general.

Having default be true is not an issue to me